src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+html {
+  font-family: Roboto, serif;
+}
+
+body {
+  margin: $margin;
```

Comment:
The variable `$margin` is used here, but it is not defined in this file. Ensure that it is defined in a separate SCSS file or imported correctly.